src/mono/wasm/build/WasmApp.targets

```diff
@@ -23,7 +23,7 @@
       - $(WasmProfilers)     - Profilers to use
       - $(AOTMode)           - Defaults to `LLVMOnly`
       - $(AOTProfilePath)    - profile data file to be used for profile-guided optimization
-      - $(WasmInvariantGlobalization)
+      - $(WasmInvariantGlobalization)       - Whenever to disable ICU. Defaults to false.
```

Comment:
Why do we need extra property? Can we use only InvariantGlobalization to avoid problems when someone sets `InvariantGlobalization = false` and `WasmInvariantGlobalization = true` ?